README.md

```diff
@@ -71,6 +71,7 @@ some text
     - pet name (e.g. `capital-mosquito.txt`)
     - alphanumeric string (e.g. `yB84D2Dv.txt`)
     - random suffix (e.g. `file.MRV5as.tar.gz`)
+    - no extension
```

Comment:
This section of the docs was for listing the specific randomization features so let's not mention this here.

```suggestion
```